src/components/breadcrumbs/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import {
+  BreadcrumbItem as ChakraBreadcrumbItem,
+  BreadcrumbItemProps as ChakraBreadcrumbItemProps,
+  Hide,
+} from '@chakra-ui/react';
+
+const BreadcrumbItem: FC<PropsWithChildren<ChakraBreadcrumbItemProps>> = ({
+  children,
+  ...itemProps
+}) => {
+  return (
+    <Hide below="sm">
```

Comment:
I think we should not hardcode this in the breadcrumb what if we have a case where we wanna show breadcrumbs on smaller devices? Or create a specific variation of the component if we wanna hide it here